examples/common_vmseries/README.md

```diff
@@ -897,129 +909,121 @@ Default value: `map[]`
+  **Note!** \
+  The `disable_password_authentication` property is by default `false` in this example. When using this value, you don't have
+  to specify anything but you can still additionally pass SSH keys for authentication. You can however set this property to 
+  `true`, then you have to specify `ssh_keys` property.
 
-- `vnet_key`        - (`string`, required) a key of a VNET defined in `var.vnets`. This is the VNET that hosts subnets used to
-                      deploy network interfaces for deployed VM.
+  For all properties and their default values see [module's documentation](../../modules/vmseries/README.md#authentication).
 
-- `interfaces`      - (`list`, required) configuration of all network interfaces
-  
-    **Note!** \
-    Order of the interfaces does matter - the 1<sup>st</sup> interface is the management one. 
+- `image`           - (`map`, required) properties defining a base image used by the deployed VM. The `image` property is
+                      required but there are only 2 properties (mutually exclusive) that have to be set, either:
 
-    For details on available properties please see [module's documentation](../../modules/panorama/README.md#interfaces).
+  - `version`   - (`string`, optional) describes the PAN-OS image version from Azure Marketplace.
+  - `custom_id` - (`string`, optional) absolute ID of your own custom PAN-OS image.
 
-    The most important ones are listed below:
+  For details on all properties refer to [module's documentation](../../modules/vmseries/README.md#image).
 
-    - `name`                    - (`string`, required) name of the network interface (will be prefixed with `var.name_prefix`).
-    - `subnet_key`              - (`string`, required) a key of a subnet to which the interface will be assigned as defined in
-                                  `var.vnets`. Key identifying the VNET is defined in `virtual_machine.vnet_key` property.
-    - `create_public_ip`        - (`bool`, optional, defaults to `false`) create a Public IP for an interface.
-    - `load_balancer_key`       - (`string`, optional, defaults to `null`) key of a Load Balancer defined in `var.loadbalancers`
-                                  variable, network interface that has this property defined will be added to the Load
-                                  Balancer's backend pool.
-    - `application_gateway_key` - (`string`, optional, defaults to `null`) key of an Application Gateway defined in `var.appgws`
-                                  variable, network interface that has this property defined will be added to the Application
-                                  Gateway's backend pool.
+- `virtual_machine` - (`map`, optional, defaults to module default) a map that groups most common VM configuration options. 
+                      Most common properties are:
 
+  - `size`              - (`string`, optional, defaults to module default) Azure VM size (type). Consult the *VM-Series
```

Comment:
the indentation for descriptions starting with `-` doesn't seem to match with virtual_machine and size lines

Reply:
Because `size` is one level down under `virtual machine`. The indentation match for all properties which are on the same level in the "tree". I opened this README in Preview mode and it looks correct.